webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -47,15 +47,16 @@ const ExperimentDetailsComp: React.FC = () => {
   const [ showCheckpoint, setShowCheckpoint ] = useState(false);
   const [ forkModalVisible, setForkModalVisible ] = useState(false);
   const [ forkModalConfig, setForkModalConfig ] = useState('Loading');
-  const [ experimentResponse, triggerExperimentRequest ] =
-    useRestApi<ExperimentDetailsParams, ExperimentDetails>(getExperimentDetails, { id });
   const storage = useStorage(STORAGE_PATH);
   const initLimit = storage.getWithDefault(STORAGE_LIMIT_KEY, MINIMUM_PAGE_SIZE);
   const initSorter: ApiSorter | null = storage.get(STORAGE_SORTER_KEY);
   const [ pageSize, setPageSize ] = useState(initLimit);
   const [ sorter, setSorter ] = useState<ApiSorter | null>(initSorter);
+  const [ experiment, setExperiment ] = useState<ExperimentDetails>();
+  const [ isLoading, setIsLoading ] = useState(true);
```

Comment:
nit: name it so it shows what this isLoading is tied to if we could have multiple of these. same with `source` for cancel token and error.

could we package these into a single struct? (would looks close to the type useRestApi is using)